src/metabase/sync/analyze.clj

```diff
@@ -54,16 +54,26 @@
 ;; newly re-fingerprinted Fields, because we'll know to skip the ones from last time since their value of
 ;; `last_analyzed` is not `nil`.
 
+(s/defn ^:private update-last-analyzed!
```

Comment:
Previously we would issue one of these updates for each table. This just switches us to issuing 1 updated for all tables.